vault_example.yml

```diff
@@ -0,0 +1,48 @@
+hello: ENC[AES256_GCM,data:1DWfYHFHwafRZSOSuMy8rHvF3Da+9NFjcwPdFPmepZl2CKHL76L38CC5H2NzIQ==,iv:AhSDXStvMXp6ulPwFWW8+izJo6l5Zem9Xh6+I7/GKys=,tag:+nkbrSzLFm1VXiggb5GknA==,type:str]
```

Comment:
What is this file here for?

Reply:
As an example, for consistency with all the other _example.yml files we have for all stores.

Reply:
Actually, my bad, this is not a store, hah. Yeah, I don't think we need this example file. The docs should be enough.